neo/bin/prompt.py

```diff
@@ -141,8 +141,8 @@ class PromptInterface:
                 'wallet unspent (neo/gas)',
                 'wallet split {addr} {asset} {unspent index} {divide into number of vins}',
                 'wallet close',
-                'send {assetId or name} {address} {amount} (--from-addr={addr}) (--fee={priority_fee})',
-                'sendmany {number of outgoing tx} (--change-addr={addr}) (--from-addr={addr}) (--fee={priority_fee})',
+                'send {assetId or name} {address} {amount} (--from-addr={addr}) (--fee={priority_fee}) (--owners=[{addr}, {addr}, ...]) (--tx-attr=[{"usage": <value>,"data":"<remark>"},{"usage":<value>,"data":"<remark>"}, ...])',
```

Comment:
can you please use 1 example input for `owners` and `tx-attr` i.e.`(--owners=[{addr}, ...])` 
I think that should be sufficient, the command is already really long so I'd like to try and keep it to a minimum